projects/api/prisma/seed.ts

```diff
@@ -16,11 +21,26 @@ async function seed() {
   await prisma.category.createMany({
     data: categories,
   });
+
+  const suppliers: CreateSupplierDto[] = [];
+  const countryList = await prisma.country.findMany();
+
+  countryList.forEach((country) => {
+    suppliers.push({
+      email: `supplier_${country.name.toLowerCase().replace(/\s+/g, '_')}@example.com`,
+      deliveryTime: Math.floor(Math.random() * 180) + 1,
+      countryId: country.id,
+    });
+  });
+
+  await prisma.supplier.createMany({
+    data: suppliers,
+  });
 }
 
 seed()
   .catch((e) => {
-    throw new Error(e);
+    throw new Error(`Error seeding database: ${e}`);
```

Comment:
you are displaying all the exception, Shouldn't it be  e.error or something, isn't?